Source/AliveLibAE/Abe.cpp

```diff
@@ -3581,6 +3581,11 @@ void Abe::State_2_StandingTurn_451830()
 
     if (currentFrame == 4 && (sInputKey_Run_5550E8 & pressed) && (sInputKey_Right_5550D0 | sInputKey_Left_5550D4) & pressed)
     {
+
+#if ORIGINAL_GAME_FIXES
+        field_108_next_motion = 0; // OG Change - Fixes "Auto-Turn" bug
+#endif
```

Comment:
I think there should be an additional cmake option for these gameplay bugs for now - since the speed runners will want to keep some (certainty this one).
 Also there should be motion enum value for 0, it should be idle.

Reply:
I created "ORIGINAL_GAME_FIXES" in cmake.options for this bugfix. Or do you mean that you'd prefer it set off by default?

Reply:
I mean it should be possible to set it per bug, so have ORIGINAL_GAME_FIXES = all fixes and a specific options for ORIGINAL_GAME_FIX_AUTO_TURN and have it so if either is enabled then its fixed, we'd want this for all "speed run strat" fixes